composer.json

```diff
@@ -39,14 +39,14 @@
         "drupal/core": "~9.1.12 || ^9.2.4",
         "drupal/default_content": "^2",
         "drupal/diff": "^1",
-        "drupal/entity_clone": "1.0-beta4",
+        "drupal/entity_clone": "^1.0@beta",
```

Comment:
@panshulK Update the module's composer.json file as well. Ex: `modules/acquia_cms_common/composer.json`
Same for all the below modules/patches which are updated.

Reply:
Also, If we are patching any module we should pin the module version for which patch has been give otherwise that patch will start failing if something changed in the next release of the module.

Reply:
Done @chandan-singh7929.

@vishalkhode1  Updated and checked the composer.json file for all modules. The updates were only required to be made to acquia_cms_common module's composer json file.